src/global_conf.c

```diff
@@ -49,9 +50,10 @@ static int set_log_level(char *log_level)
         rte_log_set_global_level(RTE_LOG_NOTICE);
     else if (!strncmp(log_level, "INFO", strlen("INFO")))
         rte_log_set_global_level(RTE_LOG_INFO);
-    else if (!strncmp(log_level, "DEBUG", strlen("DEBUG")))
+    else if (!strncmp(log_level, "DEBUG", strlen("DEBUG"))) {
         rte_log_set_global_level(RTE_LOG_DEBUG);
-    else {
+        rte_log_set_level(RTE_LOGTYPE_USER1, RTE_LOG_DEBUG);
+    } else {
```

Comment:
Besides `RTE_LOGTYPE_USER1`, `RTE_LOGTYPE_USER2` and `RTE_LOGTYPE_USER3` are also used by dpvs.
Maybe a more feasible solution is to use `rte_log_set_level_pattern` instead.
```
rte_log_set_level_pattern("user[0-9]", RTE_LOG_DEBUG)
```